Samples.Objc/Samples.ObjC/ViewControllers/CheckoutViewController/CheckoutViewController.m

```diff
@@ -0,0 +1,402 @@
+//
+//  CheckoutViewController.m
+//  PayPalNativeCheckoutObjC
+//
+//  Created by Nguyen, The Nhat Minh on 3/9/21.
+//  Copyright © 2020 PayPal. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "CheckoutViewController.h"
+#import "AddItemViewController.h"
+#import "AddItemCell.h"
+#import "ItemCell.h"
+#import "TotalCell.h"
+#import "FetchAccessTokenEndpoint.h"
+#import "CreateOrderEndpoint.h"
+#import "AccessTokenResponse.h"
+#import "CreateOrderResponse.h"
+#import "PayPalAPI.h"
+@import PayPalCheckout;
+
+@interface CheckoutViewController ()
+@property (nonatomic) UISegmentedControl *segmentedControl;
+@property (nonatomic) UITableView *tableView;
+@property (nonatomic) UIButton *checkoutButton;
+@property (nonatomic) NSMutableArray<PPCPurchaseUnitItem *> *items;
+@end
+
+@implementation CheckoutViewController
+
+// MARK: - Lifecycle
+
+- (void)viewDidLoad {
+  [super viewDidLoad];
+  [self setupSampleItem];
+  [self setupUI];
+  [self setupTableView];
+  [self setupConstraints];
+}
+
+// MARK: - PayPalCheckout
+
+- (void)startCheckout {
+  [PPCheckout startWithPresentingViewController:self
+                                    createOrder:^(PPCCreateOrderAction *action) {
+    switch (self.segmentedControl.selectedSegmentIndex) {
+      case 0:
+        // Checkout with order
+        [self createOrderCallbackCreateOrderWithAction:action];
+        break;
+      case 1:
+        // Checkout with orderID/ECToken/payToken
+        [self createOrderCallbackSetOrderIDWithAction:action];
+        break;
+      default:
+        break;
+    }
+  }
+                                      onApprove:^(PPCApproval *approval) {
+    [self onApproveCallbackWithApproval:approval];
+  }
+                                       onCancel:^{
+    [self onCancelCallback];
+  }
+                                        onError:^(PPCErrorInfo *errorInfo) {
+    [self onErrorCallbackWithErrorInfo:errorInfo];
+  }];
```

Comment:
It will look like this:
![Screen Shot 2021-04-07 at 11 13 33 AM](https://user-images.githubusercontent.com/80476442/113890592-4d219d00-9792-11eb-8165-eb87f5bb4583.png)

Coding style is not very consistent with other function calls, but I think at least for function with callbacks like `start`, this definitely looks cleaner and easier to follow :smile: